de.metas.business/src/main/java-legacy/org/compiere/model/MAllocationHdr.java

```diff
@@ -570,7 +570,9 @@ public String completeIt()
 			final int bpartnerId = line.processIt(false); // not reverse
 			bpartnerIds.add(bpartnerId);
 		}
-		updateBP(bpartnerIds);
+
```

Comment:
Can u pls point me where exactly are the stats updated when an allocation is completed?